### PR TITLE
Add 20.12.0 release and release notes.

### PR DESCRIPTION
**Pull request by putnopvut**

### CODE REVIEW

File: src/content/releases/release_20.12.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
++++
+date = 2020-07-09T08:53:23-04:00
```
Comment (dceara) [inline src/content/releases/release_20.12.0.md:2]: I guess this is "copy/paste".

File: src/content/releases/release_20.12.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- The nb\_cfg value is now propagated to the OVS database.
+- It is possible to pin ovn-northd or ovn-controller to a specific version. This can allow for less downtime during upgrades if the central components are updated before the ovn-controllers in a cluster.
+- Support for "fair" meters, which allows for multiple ACL logs to use the same meter while being rate-limited independently.
+
+### Performance Enhancements
+
+- "reject" ACLs have been reduced from generating 4 logical flows to generating 1.
+- MAC bindings are updated for directly-connected logical routers, saving the need for unnecessary ARPs/NDs.
+- The number of flows required for hairpin traffic to load balancers has been reduced.
+- ovn-northd can now group logical flows that are shared between multiple datapaths using datapath groups. This has been shown to reduce the number of logical flows in the southbound database in large deployments by a factor of 100. The option is disabled by default but likely will be switched to be enabled by default in a later release.
```
Comment (dceara) [inline src/content/releases/release_20.12.0.md:24]: `by a factor of 100` <- This is nice but I guess it depends on the number of similar logical datapaths.  Maybe we shouldn't give an absolute value here?

**Review by dceara — APPROVED** [review]
LGTM, thanks.

**Comment by dceara** [comment]
Oh, I see there's a failure in CI:
```
 Error: Error building site: unmarshal failed: Near line 1 (last key parsed 'date'): Invalid TOML Datetime: "2020-12-17T08:53:48-0500".
 ```

**Comment by putnopvut** [comment]
CI failure fixed now. I messed up the date format.

**Pull request merged**